### ISSUE TITLE
No result

### ISSUE THREAD

**Issue by josvink66** [bartmachielsen/SupermarktConnector#17]
Hi,

Is this code still working for Albert Heijn? I've installed the package and run the code. But I get no result, no error just nothing. The code I've used as test:

`from supermarktconnector.ah import AHConnector connector = AHConnector() connector.get_categories()`

Any help would be appreciated.
Greetings,
Jos

**Comment by bkiers** [comment]
@josvink66 nope, alas, it used to work but since a couple of weeks, AH changed some things and this Python library no longer works.

**Comment by rogierrijs** [comment]
It still works great. Only thing i needed to do, is change the jumbo API to v14 for now.

**Comment by lee-msr** [comment]
@rogierrijs 

How did you do it?

**Comment by gewoondemax** [comment]
@rogierrijs same question from my side ... how to do? get the scraper to work? I am totally blind in python land, help is much appreciated 
max

**Comment by rogierrijs** [comment]
Hey, in the code, an url is used, which holds the version.. like /v12/ or something.  You should change that to the one that works. 

Edit, just checked, that is for Jumbo. Not AH.

To be honest, havent used this in a long time, so maybe something else broke.
